doc-content/enterprise-only/openshift/environment-immutable-server-kjar-proc.adoc

```diff
@@ -0,0 +1,82 @@
+[id='environment-immutable-server-kjar-proc']
+// definitions for RH-SSO include
+:kieonly:
+// kieonly must be undefined at end of file
+:LOCAL_CENTRAL: {CENTRAL} Monitoring
+:modifylink: none
+= Deploying an immutable {KIE_SERVER} from KJAR services
+
+To deploy an immutable {KIE_SERVER} from KJAR services,
+ifdef::DM[]
+use the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-kieserver.yaml` template file. You can extract this file 
+endif::DM[]
+ifdef::PAM[]
+use one of the following template files:
+
+* `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-kieserver-postgresql.yaml` to use a PostgreSQL pod for persistent storage. Use this template unless there is a sufficient reason to use another template.
+
+* `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-kieserver-mysql.yaml` to use a MySQL pod for persistent storage. 
+
+* `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-kieserver-externaldb.yaml` to use an external database server for persistent storage.
++
+IMPORTANT: The standard {KIE_SERVER} image for an external database server includes drivers for MySQL and PostgreSQL external database servers. If you want to use another database server, you must build a custom {KIE_SERVER} image. For instructions, see <<externaldb-build-proc>>.
+
+You can extract these template files 
+endif::PAM[]
+from the `{PRODUCT_FILE}-openshift-templates.zip` product deliverable file. You can download the file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page.
+
+In this method of deployment, the {KIE_SERVER} retrieves the KJAR files for any services that must run on it during the startup of the pod.
```

Comment:
This sentence still seems quite complicated. Wouldn't it be better to put there something like "In this method of deployment, the {KIE_SERVER} retrieves all the required KJAR files during the startup of the pod."?